captain/navigation.go

```diff
@@ -110,7 +110,7 @@ findCandidates:
 func connectToHomeHub(ctx context.Context, dst *hub.Hub) error {
 	// Set and clean up exceptions.
 	setExceptions(dst.Info.IPv4, dst.Info.IPv6)
-	defer setExceptions(nil, nil)
+	// defer setExceptions(nil, nil)
```

Comment:
If we don't reset the exceptions after connecting, the connections to the node will no go through the SPN. I think this is guarded to only apply to Portmaster itself, so I guess this might never happen anyway.

Was this change on purpose? (As it is only commented out, not removed.)
If so, please elaborate.

Reply:
Yes this was intentional. The reason why it is commented is that I didn't understand the reason behind it, will it break something if we don't reset it? It is used only in `checkTunneling` function.

I removed it because after the changes to the firewall now it is possible to run a firewall check to a connection multiple times and change the verdict. If the info for the spn connection is not always present we try to rout the connection through the SPN.